document/FileFormat.mdk

```diff
@@ -1164,7 +1201,7 @@ necessarily contain enough information to do so.
 ### `offset` property { #file-offset }
 
 Depending on the circumstances, a `file` object either shall, may, or shall not contain a property named `offset`
-whose value is an integer.
+whose value is a non-negative integer.
```

Comment:
Yes, we have Microsoft/sarif-sdk#484, "Update schema per constraints review".

---
In reply to: [75182354](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75182354) [](ancestors = 75182354)